Initial Draft with Basic Use and Risk Description (#1)
lsst/rtn-051: breakdown-guides.rst

Impact Severity
	Automatically generated category based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

	See :ref:`Risk-Impact-Table` for categories.

Impact Score
	Automatically generated value based on ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Likelihood Score
	Automatically generated value based on ``Likelihood``.

Probability
	Automatically generated value based on ``Likelihood``.

Initial Risk Score
	Automatically generated value based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Minimum Delay (Months)
	Minimum delay if risk is realized.

Maximum Delay (Months)
	Maximum delay if risk is realized.

Likely Delay (Months)
	Most likely delay if risk is realized.

Expected Schedule Delay (Months)
	Automatically generated value based on ``Minimum Delay``, ``Maximum Delay``, and ``Likely Delay``.

.. todo::
   Define formula for Expected Schedule Delay.

Impact Time
	Date when realized risk would impact the schedule.

Impacted Event/Milestone
	The event or milestone of a realized risk.

Basis of Estimate
	Reference to basis of estimate capturing impact of realized risk.

Minimum Cost (US Dollars)

> 	Minimum cost of realized risk.

Reviewer: ```suggestion
	Minimum annual cost of realized risk.
```

Author: Added to next commit.

Maximum Cost (US Dollars)

> 	Maximum cost of realized risk.

Reviewer: ```suggestion
	Maximum annual cost of realized risk.
```

Author: Definition now combined with minimum cost.

Likely Cost (US Dollars)

> 	Likely cost of realized risk.

Reviewer: ```suggestion
	Likely annual cost of realized risk.
```

Author: Definition now combined with minimum cost.

> Costs should be estimated as they would occur, i.e. summed over the likely time period of impact (and in approximate then-year dollars). Cost estimates need only be precise to the nearest $1M (although higher precision is appreciated). This is because your cost estimates will be multiplied by your likelihood estimates, and these are likely to be uncertain by at least a factor of two.

Reviewer: ```suggestion
Costs should be estimated as they would occur, i.e. on an approximate, time-averaged, annual basis over the likely time period of impact (and in approximate then-year dollars). This is because Rubin needs to know how much funding to hold in reserve _each year_ in order to address risks as they are realized.

Cost estimates need only be precise to the nearest $1M (although higher precision is appreciated). This is because your cost estimates will be multiplied by your likelihood estimates, and these are likely to be uncertain by at least a factor of two.
```

Author: Added to next commit.

.. figure:: /_static/Risk-Example-Risk-Identification.png
    :name: Risk-Example-Risk-Identification

    Risk Identification section using an example risk.

Project
	Always use ``Rubin Operations``.

Risk ID
	Automatically generated unique identifier.

Risk Type
	``Threats`` or ``Opportunities``.

Status
	``Candidate``, ``Active``, ``Retire``, ``Realized``, or ``Depreciated``.

Risk Department
	Rubin Observatory Department which owns the risk and responsible for its management.

Risk Category and Sub Category
	Categorizes the risk using the information in :ref:`Risk-Category-Table`.
	This information is defined within the Risk Tool --- click the information button next to the field to display the information.

Risk Title
	Short, descriptive title for the risk.

Risk Statement
    All risk statements are required to conform to IF-THEN statements.
    The statement should present the possible risk event or condition ("if") and the potential outcome or consequences ("then").

Date Entered, Date Last Modified, Last Modified By
	These fields are automatically generated.

Share Risk Externally

> 	``Yes`` or ``No`` depending on if the risk is shared external to the project.

Reviewer: ```suggestion
	``Yes`` or ``No`` depending on if the risk is shared external to Rubin Operations.
```

Author: Added to next commit.

> .. todo::
>    Determine if this is external to project or external stakeholders.

Reviewer: ```suggestion
```

Author: Added to conf.py the option to suppress todos in html output for sphinx.ext.todo.

.. _Breakdown-Risk-Initial-Impact:

Risk initial impact
-------------------

Risks are analyzed by the ``Cost Impact``, ``Schedule Impact`` to the project, and the ``Likelihood`` for it to be realized.
The risk should first be analyzed under the initial condition of realization, i.e., before responses take effect.
The impact categorizations will automatically generate the :ref:`Risk Score <Breakdown-Risk-Score-Quantitative>` fields as the product of the impact and likelihood.

These are categorized into five levels of severity, as defined in :ref:`Breakdown-Risk-Tool-Tables`.
This information is defined within the Risk Tool --- click the information button next to the field to display the information.

.. figure:: /_static/Risk-Example-Analyze-Risk-Impacts.png
    :name: Risk-Example-Analyze-Risk-Impacts

    Risk Impacts sections using an example risk.

Overall Impact
	This is an optional field to categorize the overall impact of the risk to the project.

	It can increase the ``Impact Severity``, as shown in this example risk.

	See :ref:`Risk-Impact-Table` for categories.

Cost Impact
	Categorization of cost impact to the project.

> 	For the Rubin Observatory, the annual budget of $70,000,000 will be used.

Reviewer: ```suggestion
	Categorization of cost impact.

	For Rubin Observatory, the annual operating budget of $70,000,000 will be used as the reference "FY Baseline". Cost impacts are categorized relative to this annual baseline, even though in practice the cost of the realized risk may be felt (and accumulate) over multiple years.  
```

Author: Added to next commit, with modification to wording and placement.

	See :ref:`Risk-Impact-Table` for categories.

Schedule Impact
	Categorization of schedule impact to the project.

> 	Contact your department's Associate Director to determine specifics about the schedule and critical path.

Reviewer: ```suggestion
	You should discuss specifics of schedule impacts with your department's Associate Director. Some delays to the Observatory's operations can be absorbed into the data release cycle, while others can only be addressed by extending the LSST survey. The latter impacts are the ones that are most important to capture. 
```

Author: Added to next commit, with modification to wording and placement.

	See :ref:`Risk-Impact-Table` for categories.

Likelihood
	Overall chance of risk being realized.

	See :ref:`Likelihood-Table` for categories.

Existential Risk
	Select ``Yes`` if the risk is existential to NOIRLab.

> 	The Rubin Observatory Risk and Opportunity Board will determine if this is appropriate.

Reviewer: ```suggestion
	The Rubin Observatory Risk and Opportunity Board will determine if this is appropriate. You should make an initial assessment for the Risk Board to review.
```

Author: Added to next commit, with modification to placement.

Schedule/Cost Impact Description
	Use these sections to describe and comment on impact categorization and the decision on rating.

.. _Breakdown-Risk-Score-Quantitative:

Risk score and quantitative analysis 
------------------------------------

The fields under ``Risk Score`` are automatically generated based on input selections from :ref:`risk impacts <Breakdown-Risk-Initial-Impact>`.
Categories and values are based on the :ref:`Breakdown-Risk-Tool-Tables`.

> The ``Analyze Risk Quantitative`` section will not affect values and categories; however, the section will record the impact justification and provide information needed to categorize the impacts.

Reviewer: ```suggestion
The ``Analyze Risk Quantitative`` section will not affect values and categories; however, the section will record the impact justification and provide information needed to categorize the impacts. In practice you may want to go back and adjust your impact categorizations after completing the the ``Analyze Risk Quantitative`` section.
```

Author: Added to next commit, with modification to wording.

.. figure:: /_static/Risk-Example-Analyze-Risk-Score-and-Quantitative.png
    :name: Risk-Example-Analyze-Risk-Score-and-Quantitative

    Risk Score and Analysis Quantitative sections using an example risk.

Impact Severity
	Automatically generated category based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

	See :ref:`Risk-Impact-Table` for categories.

Impact Score
	Automatically generated value based on ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Likelihood Score
	Automatically generated value based on ``Likelihood``.

Probability
	Automatically generated value based on ``Likelihood``.

Initial Risk Score
	Automatically generated value based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Minimum Delay (Months)

> 	Minimum delay if risk is realized.

Reviewer: ```suggestion
	Minimum required extension to the LSST survey if risk is realized.
```

Reviewer: @DukeofSuffolk I think you shoudl ignore this previous review comment of mine (and others like it). Looking at the risks that have been analyzed, most do not lead to a needed extension to the survey - and most owners specify the impacted milestone (eg DR1, Start of Operations, etc) so it's straightforward to see which risks would affect the survey length. So, the original wording you had here was better: generic minimum delay if risk is realized.

Maximum Delay (Months)

> 	Maximum delay if risk is realized.

Reviewer: ```suggestion
	Maximum required extension to the LSST survey if risk is realized.
```

Reviewer: Ignore this suggestion (see above).

Likely Delay (Months)

> 	Most likely delay if risk is realized.

Reviewer: ```suggestion
	Most likely required extension to the LSST survey if risk is realized.
```

Reviewer: Ignore this suggestion (see above).

Expected Schedule Delay (Months)
	Automatically generated value based on ``Minimum Delay``, ``Maximum Delay``, and ``Likely Delay``.

.. todo::
   Define formula for Expected Schedule Delay.

Impact Time
	Date when realized risk would impact the schedule.

Impacted Event/Milestone
	The event or milestone of a realized risk.

Basis of Estimate
	Reference to basis of estimate capturing impact of realized risk.

Minimum Cost (US Dollars)
	Minimum cost of realized risk.

Maximum Cost (US Dollars)
	Maximum cost of realized risk.

Likely Cost (US Dollars)
	Likely cost of realized risk.

.. todo::

>    Define when dollar value should be evaluated, e.g., year of impact, year of entry.

Reviewer: ```suggestion
Costs should be estimated as they would occur, i.e. summed over the likely time period of impact (and in approximate then-year dollars). Cost estimates need only be precise to the nearest $1M (although higher precision is appreciated). This is because your cost estimates will be multiplied by your likelihood estimates, and these are likely to be uncertain by at least a factor of two.
```

Expected Cost Impact
	Automatically generated value based on the following formula:

	``Minimum Cost`` + ``Likely Cost``*4 + ``Maximum Cost``/6

Expected Monetary Value
	Automatically generated value based on the following formula:

	(``Minimum Cost`` + ``Likely Cost``*4 + ``Maximum Cost``/6) * ((``Likelihood Score`` * 0.20) - 0.10)

Financial Provision
	This field is not used by Rubin Observatory.

Number of Possible Occurrences
	Define the number of potential occurrences this risk can be realized.

.. _Breakdown-Risk-Residual-Impact:

Risk residual impact
--------------------

.. Use :ref:`Response Plans <Breakdown-Plan>` when section is ready.

> After a risk is realized, Response Plans are used to address it.

Reviewer: ```suggestion
After a risk is identified, Response Plans are used to address it.
```

Reviewer: This needs checking, but the fact that a Response Plan can be categorized as "Mitigate" suggests that the residual risk is what is left after eg mitigation, and it may yet not be realized. I think the residual cost impact is the cost to Rubin of addressing the risk if it is realized, after the response plan (eg mitigation) has been executed. What do you think?

Author: I agree with the definition of residual cost impact you state here. I think it follows that this suggested change is appropriate. Added to next commit.

.. figure:: /_static/Response-Plan-Types.png
    :name: Response-Plan-Types

    Four processes and respective threat or opportunity response plan types.

Response Types for Threats
	Avoid
		Changing your strategy or plans to avoid the risk.
		This risk response strategy is about removing the threat by any means.
		That can mean changing your management plan to avoid the risk because it’s detrimental to the project/program.

	Transfer
		Passing ownership and/or liability to a third party to resolve the risk, e.g., purchase fire insurance for an unfinished building.

	Mitigate
		Reducing the probability and/or impact of the risk below a threshold of acceptability.
		Some risks cannot be avoided and need to take action to reduce the impact of the risk, e.g., work procedures and equipment designed to reduce workplace safety risks.

	Accept
		Recognizing residual risks and devising responses to control and monitor them.
		This risk response strategy consists of identifying a risk and documenting all the risk management information about it, but not taking any action unless the risk is realized.

> 

Reviewer: ```suggestion

Note that a risk may still be realized after either Mitigating it or Accepting it; the difference in these two responses before the risk is realized can be summarized as "do something" or "do nothing." 

```

Author: Added to next commit, with modification to wording and placement. Please see if this captures what you wanted though because I wasn't sure what you were getting at besides an example that these things are context-dependent.

Response Types for Opportunities
	Exploit
		Exploiting a risk to make use of the opportunity that becomes available if that risk occurs.

	Share
		Distributing the risk across multiple stakeholders (teams/projects/programs).

	Enhance
		An action that is taken to increase the chance of the opportunity occurring.

	Ignore
		Opportunities that cannot be actively addressed through other opportunity response types can be ignored, with no special measures being taken to address them.

Escalation
	Risks can be escalated to NOIRLab Directorate or other program/services for their attention.

Related Response Plans
	Lists Response Plans associated with this risk.

Related Actions

> 	Lists Actions associated with this risk.

Reviewer: ```suggestion
	Lists Actions associated with this risk. (Actions are what you do if the risk is realized.)
```

Author: Added to next commit.

Author: Added to next commit, with modification to wording and placement.

Residual Overall Impact

> 	This is an optional field to categorize the overall impact of the risk to the project after the response plan is in effect.

Reviewer: ```suggestion
	This is an optional field to categorize the overall impact of the risk to Rubin Operations after the response plan is in effect.
```

Author: Added to next commit.

	It can increase the Impact Severity, as shown in this example risk.

	See :ref:`Risk-Impact-Table` for categories.

Residual Cost Impact

> 	Categorization of cost impact to the project after the response plan is in effect.

Reviewer: ```suggestion
	Categorization of cost impact to Rubin Operations after the response plan is in effect.
```

Author: Added to next commit.

> 	For the Rubin Observatory, the annual budget of $70,000,000 will be used.

Reviewer: ```suggestion
	Again, the annual operating budget of Rubin Observatory of $70,000,000 is used as the FY baseline reference, against which cost impacts will be compared.
```

Author: Added to next commit, using same language as `cost impact`.

	See :ref:`Risk-Impact-Table` for categories.

Residual Likelihood
	Overall chance of risk being realized after the response plan is in effect.

	See :ref:`Likelihood-Table` for categories.

Residual Schedule Impact
	Categorization of schedule impact to the project after the response plan is in effect.

> 	Contact your department’s Associate Director to determine specifics about the schedule and critical path.

Reviewer: ```suggestion
	Contact your department’s Associate Director to determine specifics about the schedule impacts, and in particular whether an extension to the LSST survey would be needed as an Action if the risk was realized.
```

Author: Added to next commit, and copied language to `schedule impact`.

The impact categorizations will automatically generate the :ref:`Risk Score <Breakdown-Risk-Score-Quantitative>` fields as the product of the impact and likelihood.

These are categorized into five levels of severity, as defined in :ref:`Breakdown-Risk-Tool-Tables`.
This information is defined within the Risk Tool --- click the information button next to the field to display the information.

.. figure:: /_static/Risk-Example-Analyze-Risk-Impacts.png
    :name: Risk-Example-Analyze-Risk-Impacts

    Risk Impacts sections using an example risk.

Overall Impact
	This is an optional field to categorize the overall impact of the risk to the project.

	It can increase the ``Impact Severity``, as shown in this example risk.

	See :ref:`Risk-Impact-Table` for categories.

Cost Impact
	Categorization of cost impact, relative to the Rubin Observatory ``FY Baseline`` operating budget of $70,000,000.

	Cost impacts are categorized relative to the annual baseline, even though in practice the cost of the realized risk may be felt and/or accumulated over multiple years.

	See :ref:`Risk-Impact-Table` for categories.

Schedule Impact

> 	Categorization of schedule impact, relative to the critical path of the Rubin Observatory's schedule (e.g., the data release cycle).

Reviewer: ```suggestion
	Categorization of schedule impact, relative to the critical path of the Rubin Observatory's schedule (e.g., the data release cycle, the summit maintenance schedule, the start of operations, or the completion of the LSST survey).
```

Author: Added to next commit.

    :name: Risk-Example-Analyze-Risk-Score-and-Quantitative

    Risk Score and Analysis Quantitative sections using an example risk.

Impact Severity
	Automatically generated category based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

	See :ref:`Risk-Impact-Table` for categories.

Impact Score
	Automatically generated value based on ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Likelihood Score
	Automatically generated value based on ``Likelihood``.

Probability
	Automatically generated value based on ``Likelihood``.

Initial Risk Score
	Automatically generated value based on ``Overall Impact``, ``Cost Impact``, ``Schedule Impact`` and ``Likelihood``.

Minimum Delay (Months)
	Minimum delay if risk is realized.

Maximum Delay (Months)
	Maximum delay if risk is realized.

Likely Delay (Months)
	Most likely delay if risk is realized.

Expected Schedule Delay (Months)
	Automatically generated value based on ``Minimum Delay``, ``Maximum Delay``, and ``Likely Delay``.

.. todo::
   Define formula for Expected Schedule Delay.

Impact Time
	Date when realized risk would impact the schedule.

Impacted Event/Milestone

> 	The event or milestone of a realized risk.

Reviewer: ```suggestion
	The event or milestone of a realized risk. This is important, so that the meaning of the schedule delay is clear. Choose from: LSST Survey Start, DR1, DR2, Year 1 Annual Maintenance, ... DR11, LSST Survey Finish.
```

Author: Added to next commit. Because you seem to indicate a full list of milestones to choose from, I've included a todo item to list out the milestones.

.. figure:: /_static/Response-Plan-Types.png
    :name: Response-Plan-Types

    Four processes and respective threat or opportunity response plan types.

Response Types for Threats
	Avoid
		Changing your strategy or plans to avoid the risk.
		This risk response strategy is about removing the threat by any means.
		That can mean changing your management plan to avoid the risk because it’s detrimental to the project/program.

	Transfer
		Passing ownership and/or liability to a third party to resolve the risk, e.g., purchase fire insurance for an unfinished building.

	Mitigate
		Reducing the probability and/or impact of the risk below a threshold of acceptability.
		Some risks cannot be avoided and need to take action to reduce the impact of the risk, e.g., work procedures and equipment designed to reduce workplace safety risks.

	Accept
		Recognizing residual risks and devising responses to control and monitor them.
		This risk response strategy consists of identifying a risk and documenting all the risk management information about it, but not taking any action unless the risk is realized.

> Response Types for Opportunities

Reviewer: ```suggestion
Some risks may have response plans of more than one type. In this case, specify plan type of the costliest present. The plan types in order of increasing costliness are: Accept (cheapest), Transfer, Avoid, Mitigate (most expensive). 

Response Types for Opportunities
```

Author: Added to next commit. We should state what the order is for opportunities (added todo item).
